test/multinetwork.jl

```diff
@@ -270,16 +270,37 @@ TESTLOG = Memento.getlogger(PowerModels)
             end
         end
 
+        @testset "test ac polar mi opf" begin
+            result = PowerModels._run_mn_strg_mi_opf(mn_data, PowerModels.ACPPowerModel, juniper_solver)
+
+            @test result["termination_status"] == LOCALLY_SOLVED
+            @test isapprox(result["objective"], 70435.9; atol = 1e0)
+
+            @test isapprox(sum(network["storage"]["1"]["ps"] for (n, network) in result["solution"]["nw"]), -0.178962; atol=1e-3)
+            @test isapprox(sum(network["storage"]["1"]["qs"] for (n, network) in result["solution"]["nw"]), -0.078913; atol=1e-3)
+
+            @test isapprox(sum(network["storage"]["2"]["ps"] for (n, network) in result["solution"]["nw"]), -0.238266; atol=1e-3)
+            @test isapprox(sum(network["storage"]["2"]["qs"] for (n, network) in result["solution"]["nw"]),  0.000000; atol=1e-3)
+        end
+
         @testset "test dc polar opf" begin
             result = PowerModels._run_mn_strg_opf(mn_data, PowerModels.DCPPowerModel, ipopt_solver)
 
             @test result["termination_status"] == LOCALLY_SOLVED
-            @test isapprox(result["objective"], 69706.9; atol = 1e0)
+            @test isapprox(result["objective"], 69703.07; atol = 1e0)
 
-            for (n, network) in result["solution"]["nw"]
-                @test isapprox(network["storage"]["1"]["ps"], -0.0447992; atol = 1e-3)
-                @test isapprox(network["storage"]["2"]["ps"], -0.0596441; atol = 1e-3)
-            end
+            @test isapprox(sum(network["storage"]["1"]["ps"] for (n, network) in result["solution"]["nw"]), -0.180000; atol=1e-3)
+            @test isapprox(sum(network["storage"]["2"]["ps"] for (n, network) in result["solution"]["nw"]), -0.240000; atol=1e-3)
+        end
+
+        @testset "test dc polar mi opf" begin
+            result = PowerModels._run_mn_strg_mi_opf(mn_data, PowerModels.DCPPowerModel, juniper_solver)
```

Comment:
The cbc solver should work here, correct?

Reply:
the objective is quadratic.  Should I create a new storage matpower without a quadratic objective?

Reply:
Making the objective linear makes this test return `NUMERICAL_ERROR`
https://github.com/lanl-ansi/PowerModels.jl/blob/cd543a09e2b8de5c2d2aee2059a64a03b9263b07/test/multinetwork.jl#L274

I will change `case5_strg.m` to have linear gen_cost, and make this test alter the gen_cost data to run properly.